tests/test_pipeline_images.py

```diff
@@ -11,7 +11,6 @@
 
 from scrapy.http import Request, Response
 from scrapy.item import Field, Item
-from scrapy.pipelines.images import ImagesPipeline
```

Comment:
I think it's no longer needed to more this import as that file doesn't import PIL anymore?